instance/models/mixins/openedx_theme.py

```diff
@@ -117,12 +127,94 @@ def get_theme_settings(self):
             "EDXAPP_DEFAULT_SITE_THEME": "simple-theme",
             "SIMPLETHEME_EXTRA_SASS": """
                 .global-header {{
-                    background: {header_bg};
+                    background: {header_bg}; // Header color
```

Comment:
@giovannicimolin, is it possible to avoid adding all the SASS/CSS code in Ocim and instead use it from an external skeleton theme? That way, Ocim code will be cleaner and separate.